README.md

```diff
@@ -81,12 +81,32 @@ These can be viewed at https://app.diagrams.net/
 
 You should see output similar to the following:
 ```bash
- * Serving Flask app "app" (lazy loading)
+ * Serving Flask app "todo_app/todo_app/app" (lazy loading)
  * Environment: development
  * Debug mode: on
  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
  * Restarting with fsevents reloader
  * Debugger is active!
  * Debugger PIN: 226-556-590
 ```
-Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
\ No newline at end of file
+Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+
+### Heroku deployment
+Commands to build and deploy to heroku
+```
+
+docker build --target test --tag my-test-image .
+docker run my-test-image todo_app/test_tasks.py
+docker run -e TRELLO_KEY=$TRELLO_KEY -e TRELLO_TOKEN=$TRELLO_TOKEN -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID -e 
+    TRELLO_TODO_LIST_ID=$TRELLO_TODO_LIST_ID -e TRELLO_DOING_LIST_ID=$TRELLO_DOING_LIST_ID -e TRELLO_DONE_LIST_ID=$TRELLO_DONE_LIST_ID my-test-image todo_app/test_client.py
+docker run -e TRELLO_KEY=$TRELLO_KEY -e TRELLO_TOKEN=$TRELLO_TOKEN -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID -e       
+    TRELLO_TODO_LIST_ID=$TRELLO_TODO_LIST_ID -e TRELLO_DOING_LIST_ID=$TRELLO_DOING_LIST_ID -e TRELLO_DONE_LIST_ID=$TRELLO_DONE_LIST_ID   my-test-image todo_app/test_system.py
+docker build --target prod --tag $DOCKER_USER/todo-app:latest .  
+docker push $DOCKER_USER/todo-app:latest
+docker tag $DOCKER_USER/todo-app:latest registry.heroku.com/$HEROKU_APP/web
+docker push registry.heroku.com/$HEROKU_APP/web
+heroku login
+heroku container:release web --app=todo-app-pv
```

Comment:
It's not especially helpful to repeating a big block of commands that is in .travis.yml
This just creates something you have to keep up to date. 

You haven't clarified anything because these aren't just "Commands to build and deploy to heroku" as you claim. This also builds and runs tests, totally unrelated to the deployment. And it requires having a bunch of environment variables set up that you haven't explained.

You could keep the last few commands if you think manual deployments would ever useful or it's helpful to note because Heroku's documentation is unclear.